package/com.unity.formats.usd/Tests/Editor/ImportHelpersTests.cs

```diff
@@ -4,95 +4,129 @@
 using UnityEngine;
 using USD.NET;
 using USD.NET.Unity;
+using System.Linq;
 using Object = UnityEngine.Object;
+using System;
 
 namespace Unity.Formats.USD.Tests
 {
     public class ImportHelpersTests : BaseFixtureEditor
     {
-        Scene CreateTestAsset(string fileName)
+        Scene m_scene;
+
+        [SetUp]
+        public void CreateTestScene()
+        {
+            m_scene = CreateTestUsdScene();
```

Comment:
Should there be a corresponding TearDown for this?

Reply:
added!